### PR TITLE
Update data

### PR DESCRIPTION
**Pull request by zapbot**
From:
zaproxy/zap-admin@85b28750bc32abdc79908c045a2fa749a8fb2612

### CODE REVIEW

File: site/data/addons.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@
   version: 3
 - id: ascanrules
   name: Active scanner rules
-  description: The release status Active Scanner rules
+  description: The release quality Active Scanner rules
```
Comment (thc202) [inline site/data/addons.yaml:45]: Going to revert these changes.

Comment (thc202) [reply site/data/addons.yaml]: Done.

**Review by thc202 — APPROVED** [review]

**Review by psiinon — APPROVED** [review]
